Update Visual Studio extention documentation (#2287)
telerik/blazor-docs: getting-started/vs-integration/upgrade-wizard.md

>tip We recommend selecting all projects that reference UI for Blazor and upgrading them to the same product version.

## Upgrade Localization File

If localization is enabled in the application, you can select the **Upgrade the localization file** option. The Upgrade Wizard will add new localization keys from the default (English) localization file that UI for Blazor distributes.

> The option is selected by default if there is a **Resources\TelerikMessages.resx** file in the current project. In this case, **Resources\TelerikMessages.resx** file is selected in the file browser. 

Reviewer: File/folder paths must use the `inline code style`.
```suggestion
The option is selected by default if there is a `Resources\TelerikMessages.resx` file in the current project. In this case, `Resources\TelerikMessages.resx` file is selected in the file browser. 
```

## Create Backup

You can create a backup before running the upgrade wizard by selecting the **Create a backup before upgrading** option.

> >When the project and solution are placed in the same directory, the default backup location is set to the **%USERPROFILE%\Documents** folder.

Reviewer: ```suggestion
>When the project and solution are placed in the same directory, the default backup location is set to the `%USERPROFILE%\Documents` folder.
```

## Upgrade Log

> In case one of the options ([Create a backup before upgrading](#create-backup) or [Upgrade the localization file](#upgrade-localization-file)) is selected, an **Upgrade Log** will be opened at the end of the upgrade. The log can show the backup location, an entry for each localization key added or a localization upgrade error. Each upgrade log file is in HTML format. The file name contains the date and time of the upgrade.

Reviewer 2: You can spare the passive voice.

```suggestion
In case one of the options ([Create a backup before upgrading](#create-backup) or [Upgrade the localization file](#upgrade-localization-file)) is selected, an **Upgrade Log** will open at the end of the upgrade. The log can show the backup location, an entry for each localization key added or a localization upgrade error. Each upgrade log file is in HTML format. The file name contains the date and time of the upgrade.
```

## Troubleshooting

The **Finish** button will display a tooltip, indicating a page error when:

> * The **Upgrade the localization file** option is selected and the pointed **directory is empty** or the file **is read-only**.

Reviewer 2: We shouldn't use bold for emphasizing. There are places we have done that in the past, but it's not compliant with the Style Guide.
